test/test-tetris.js

```diff
@@ -1088,3 +1088,139 @@ describe("generation Square Tetramino", function(){
 		assert.deepEqual(transformation, bodyTetramino);	
 	})
 })
+
+describe("L-tetramino rotation 4x4", function() {
+	let field = tetris.buildField(`
+		----
+		----
+		----
+		----
+	`);
+	
+	it('After 4 turns on an empty field, the L-tetramino should remain in the same place where it was;', function(){
+		let L_Tetramino = tetris.buildTetramino(`
+			----
```

Comment:
All the parameters to buildTetromino can be 3x3 to simplify the picture - 4x4 is not necessary